libs/community/langchain_community/vectorstores/pinecone.py

```diff
@@ -347,9 +339,8 @@ def max_marginal_relevance_search(
             embedding, k, fetch_k, lambda_mult, filter, namespace
         )
 
-    @classmethod
```

Comment:
this is a breaking change for anyone using this as documented here. Is there a way we can keep this as a classmethod? https://api.python.langchain.com/en/latest/vectorstores/langchain_community.vectorstores.pinecone.Pinecone.html#langchain_community.vectorstores.pinecone.Pinecone.get_pinecone_index